### DISCUSSION TITLE
Menu item descriptions

### DISCUSSION THREAD

**Discussion by rosemaryluo** [twilio-labs/paste#2599] [Q&A] [answered]
Hi Paste team,

Looking to add some descriptions to menu items for a combobox, experimenting with a few different stylings in my figma (linked below) and wondering whether there's any recommendation from Paste

<img width="992" alt="image" src="https://user-images.githubusercontent.com/87049408/183513542-3ee827ad-c916-4ea0-9562-857006c0584e.png">

Figma: https://www.figma.com/file/rok8UhaMG22Xqck3yanEHX/Verification-and-Ahoy-migration-to-Paste-(2022)?node-id=524%3A29429

**Comment by loreina** [comment]
Hi @rosemaryluo!

I created a [mockup](https://www.figma.com/file/SpeLpBoqN2vZv8KWcSn0J1/Paste-Feedback?node-id=1719%3A470725) of what a Combobox option with a description could look like. We would recommend using the Combobox [option template](https://paste.twilio.design/components/combobox/#combobox-using-option-template) and putting the description text in `color-text-weak` below the option at the same font size. 

Hope this helps and let us know if you have any other questions!

**Comment by loreina** [comment, marked as answer]
Hi @rosemaryluo,

We published an update to the [Paste Components library](https://www.figma.com/file/E6KUvMhioUmAgN0nwmReTM/Paste-Components?node-id=14927%3A85867) to add a listbox item description variant. In code, we'd still recommend using the Combobox [option template](https://paste.twilio.design/components/combobox/#combobox-using-option-template) to compose this variant following the designs in Figma. Thanks!

**Reply by rosemaryluo** [reply]
@loreina Awesome, thanks for the update! Please let us know when the code is available as well and we can update our implementation.

**Reply by loreina** [reply]
For now, the listbox variant with and item and description will be composable with the option template. We'll consider adding it as a variant in code in the future.
